src/pid_ros.cpp

```diff
@@ -284,7 +284,7 @@ void PidROS::publish_pid_state(double cmd, double error, rclcpp::Duration dt)
       rt_state_pub_->msg_.error = error;
       rt_state_pub_->msg_.error_dot = d_error;
       rt_state_pub_->msg_.p_error = p_error;
-      rt_state_pub_->msg_.i_error = i_error;
+      rt_state_pub_->msg_.i_error = i_term;
```

Comment:
I think you said you will change the message in a different PR since this is really weighted i-error.